srcpkgs/libglvnd/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'libglvnd'
+pkgname=libglvnd
+version=1.2.0
+revision=1
+wrksrc="libglvnd-v${version}"
+build_style=gnu-configure
+hostmakedepends="autoconf libtool automake pkg-config python"
+makedepends="libXext-devel libX11-devel xorgproto"
+short_desc="GL Vendor-Neutral Dispatch library"
+maintainer="Stefano Ragni <st3r4g@protonmail.com>"
+license="custom:NVIDIA"
```

Comment:
Isn't the license MIT?

Reply:
This is a modified MIT with additional restrictions. Namely,

> Any additions, deletions, or changes to the original source files
> must be clearly indicated in accompanying documentation.
> 
> If only executable code is distributed, then the accompanying
> documentation must state that "this software is based in part on the
> work of the Khronos Group."

I don't think it's OK to just mark as MIT. I'm not sure what verbiage Void typically uses here -- maybe "Modified MIT" would be applicable. Custom by Nvidia is not inaccurate -- nobody stated it's proprietary.

Reply:
so what's the deal?

Reply:
I set this as `custom:MIT-alike` for now